gtest_parallel.py

```diff
@@ -559,7 +559,16 @@ def main():
   if options.dump_json_test_results is not None:
     test_results = CollectTestResults(options.dump_json_test_results)
 
-  save_file = os.path.join(os.path.expanduser("~"), ".gtest-parallel-times")
+  if sys.platform == 'win32':
+    default_cache_path = os.path.join(os.path.expanduser('~'), 'AppData', 'Local')
+    cache_path = os.environ.get('LOCALAPPDATA', default_cache_path)
+  else:
+    # We don't use xdg module since it's not a standard.
+    default_cache_path = os.path.join(os.path.expanduser('~'), '.cache')
+    cache_path = os.environ.get('XDG_CACHE_HOME', default_cache_path)
+
+  save_file = os.path.join(cache_path, 'gtest-parallel')
```

Comment:
I think my main concern was if .cache/ is not available (I don't think this is as standard as having a ~/ directory). Could it fall back to ~/.gtest-parallel-times as a last resort, or ignore the file and spit out some warning?